alpine/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM alpine:3.12
+LABEL maintainer="Peter Martini <PeterCMartini@GMail.com>, Zak B. Elep <zakame@cpan.org>"
+
+COPY *.patch /usr/src/perl/
+WORKDIR /usr/src/perl
+
+RUN set -x \
+    && apk add --no-cache --virtual .build-deps curl procps tar build-base make xz \
+    && curl -SL https://www.cpan.org/src/5.0/perl-5.32.0.tar.xz -o perl-5.32.0.tar.xz \
+    && echo '6f436b447cf56d22464f980fac1916e707a040e96d52172984c5d184c09b859b *perl-5.32.0.tar.xz' | sha256sum -c - \
+    && tar --strip-components=1 -xaf perl-5.32.0.tar.xz -C /usr/src/perl \
+    && rm perl-5.32.0.tar.xz \
+    && cat *.patch | patch -p1 \
+    && ./Configure -Duse64bitall -Duseshrplib  -des \
+    && make -j$(nproc) \
+    && TEST_JOBS=$(nproc) make test_harness \
+    && make install \
+    && cd /usr/src \
+    && curl -LO https://www.cpan.org/authors/id/M/MI/MIYAGAWA/App-cpanminus-1.7044.tar.gz \
+    && echo '9b60767fe40752ef7a9d3f13f19060a63389a5c23acc3e9827e19b75500f81f3 *App-cpanminus-1.7044.tar.gz' | sha256sum -c - \
+    && tar -xzf App-cpanminus-1.7044.tar.gz && cd App-cpanminus-1.7044 && perl bin/cpanm . && cd /root \
+    && runDeps="$( \
+        scanelf --needed --nobanner --recursive /usr/local \
+            | awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
+            | sort -u \
+            | xargs -r apk info --installed \
+            | sort -u \
+        )" \
+    && apk add --virtual .perl-rundeps $runDeps make wget \
```

Comment:
It seems odd to me that `make` and `wget` are included here -- are they required for successful use of Perl?  Are they both kept in the `slim` variants too?

(In Alpine at least, `wget` is provided by BusyBox in the stock image, which might be enough for the intended purpose of this line.)

Reply:
Its in the commit message:

>We previously had `PERL_CPANM_OPT=--no-wget`, but since that can be a
>user-set variable in child images, let's just install the real wget
>instead (especially since the busybox builtin isn't as feature-rich.)

Also cpanm uses GNU wget options (https://github.com/miyagawa/cpanminus/issues/598) and while I was thinking of just pulling in `curl` as with the `:slim` variants, I figured `wget` is better for this case, precisely to override the busybox builtin.

Reply:
And as for `make`, that's needed even for pure-Perl ExtUtils::MakeMaker builds (though `cpanm` can cheat somewhat if the dist follows https://github.com/Perl-Toolchain-Gang/cpan-static/blob/master/lib/CPAN/Static/Spec.pm, but still needs the a fallback for `make` regardless.)

Reply:
Doh sorry, should've checked the commit messages :see_no_evil: